### ISSUE TITLE
Incorporate new 2018.1.1 custom objects

### ISSUE THREAD

**Issue by christopherwxyz** [mark-keaton/NetSuiteSDF#8]
The following custom objects are support in 2018.1.1 sdfcli, but not supported for NetSuiteSDF VS Code plugin. 

```
advancedpdftemplate
customtransactiontype
emailtemplate
kpiscorecard
publisheddashboard
savedcsvimport
```

**Comment by JonnyBoy333** [comment]
These should be available now. #15

**Comment by mark-keaton** [comment]
Should be available in Marketplace now for 0.4.0.
